addons/mail/static/src/models/composer_view.js

```diff
@@ -445,9 +436,9 @@ registerModel({
                         this.threadView.startEditingLastMessageFromCurrentUser();
                         break;
                     }
-                    if (this.hasSuggestions) {
-                        this.setPreviousSuggestionActive();
-                        this.update({ hasToScrollToActiveSuggestion: true });
+                    if (this.composerSuggestionListView) {
```

Comment:
what prevents you from replacing `hasSuggestions` with `composerSuggestionListView` everywhere? and getting rid of `hasSuggestions` in the process?

Reply:
`hasSuggestions` is a data concept, `composerSuggestionListView` is a view concept, while there are overlaps (example in this case where I made the diff) it's not strictly always the case in theory.

All this code requires further clean up, I must made what I felt necessary to fix the identifying fields situation.